platform/ios/scripts/bazel-generate-plists.sh

```diff
@@ -82,4 +82,5 @@ cat > platform/ios/bazel/config.bzl <<EOF
 APPLE_MOBILE_PROVISIONING_PROFILE_TEAM_ID = "$teamid"
 APPLE_MOBILE_PROVISIONING_PROFILE_NAME = "iOS Team Provisioning Profile: *"
 BUNDLE_ID_PREFIX = "$bundleidprefix"
+BUILD_MODE = "xcode"
```

Comment:
Should be good to leave this out now.

Reply:
ok, I'll revert. but this way the script will produce a `config.bzl` file that won't work because of the missing `BUILD_MODE`.